tests/tests_change_indication_on_repeat_run.yml

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+- hosts: all
+  vars:
+    interface: testnic1
+    type: veth
```

Comment:
```suggestion
    type: veth
    network_connections:
      - name: testnic1
        state: up
        type: ethernet
        ip:
          address:
            - 192.0.2.2/24
```

Reply:
Placing `network_connections` in top `vars` section does not works.  How can I use it in the `import_role`?

Reply:
> Placing `network_connections` in top `vars` section does not works. How can I use it in the `import_role`?

Can you use `include_role` instead of `import_role`?

Reply:
Yes. The `include_role` did the magic.

Reply:
Thanks a lot.